server/src/test/java/org/elasticsearch/cluster/metadata/IndexMetadataVerifierTests.java

```diff
@@ -49,6 +49,42 @@ public void testArchiveBrokenIndexSettings() {
         assertSame(indexMetadata, src);
     }
 
+    public void testConvertSearchableSnapshotSettings() {
```

Comment:
I wonder if we should move this test to xpack-core, since it can then rely on the constants. At least that would surface issues if the constants are every changed, without having to make the verifier plugable?

Reply:
Makes sense, I moved the test.